package.json

```diff
@@ -2,13 +2,31 @@
     "private": true,
     "scripts": {
         "dev": "vite",
-        "build": "vite build"
+        "build": "vite build",
+        "development": "mix",
+        "watch": "mix watch",
+        "watch-poll": "mix watch -- --watch-options-poll=1000",
+        "hot": "mix watch --hot",
+        "prod": "npm run production",
+        "production": "mix --production",
```

Comment:
Are the `mix` tasks correct? Thought switching to Vite will remove those.

Reply:
Fixed!